ci/checks/headers_test.sh

```diff
@@ -10,7 +10,7 @@ DIRNAMES="cudf cudf_test"
 
 # existence tests for lib${LIBNAME}
 for DIRNAME in ${DIRNAMES[@]}; do
-    HEADERS=`cd cpp && find include/${DIRNAME}/ -type f \( -iname "*.h" -o  -iname "*.hpp" \) -printf "    - test -f \\\$PREFIX/%p\n" | sort`
+    HEADERS=`cd cpp && find include/${DIRNAME}/ -type f \( -iname "*.h" -o  -iname "*.hpp" \) -printf "        - test -f \\\$PREFIX/%p\n" | sort`
```

Comment:
Minor nit: Why the extra indentation?

Reply:
Assuming this is to address the extra indentation added to the tests through the nested structure of the updated recipe?

Reply:
> Assuming this is to address the extra indentation added to the tests through the nested structure of the updated recipe?

Nailed it!

Reply:
> Assuming this is to address the extra indentation added to the tests through the nested structure of the updated recipe?

Nailed it!